terraform/terraform.tfstate

```diff
@@ -0,0 +1,587 @@
+{
```

Comment:
Any reason we want to push `tfstate` files? Aren't these auto-generated?

Reply:
These are generated on running `terraform apply`, it is better to have it here since it shows the exact state corresponding to `main.tf`, will help the user understand the correspondence better

Reply:
[Here](https://dev.to/loujaybee/should-you-commit-the-terraform-tfstate-file-to-git-5054) and [here](https://stackoverflow.com/a/41482391) are 2 discussions telling us why we shouldn't commit `.tfstate` files in git.

Reply:
sure, removed it.